src/emulator/controller.ts

```diff
@@ -886,6 +887,21 @@ export async function startAll(
     await startEmulator(hostingEmulator);
   }
 
+  // similar to the hosting emulator, the App Hosting emulator should also
+  // start after the other emulators
```

Comment:
can you extend on why this is the case?

e.g. is it because the service running on app hosting may depend on other emulator, like Firestore?